### PR TITLE
sap_general_preconfigure: use meta/argument_specs.yml (issue #320)

### PR DESCRIPTION
**Pull request by berndfinger**
After https://github.com/sap-linuxlab/community.sap_install/issues/321 (use argument_specs and update README.md for sap_netweaver_preconfigure) is merged, the next step is to do the same for sap_general_preconfigure.

### CODE REVIEW

File: roles/sap_general_preconfigure/meta/argument_specs.yml [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+        default: "{{ ((0.75 * (ansible_memtotal_mb + ansible_swaptotal_mb)) / 1024) | round | int }}"
+        description:
+          - The size of the tmpfs in GB. The formula used here is mentioned in SAP note 941735.
+        required: false
+        type: str
+
+      sap_general_preconfigure_modify_etc_hosts:
+        default: true
+        description:
+          - Set to `no` if you do not want the role to modify the `/etc/hosts` file.
```
Comment (ja9fuchs) [inline roles/sap_general_preconfigure/meta/argument_specs.yml:236]: I assume this should be "Set to `false`..." for consistency?

Comment (berndfinger) [reply roles/sap_general_preconfigure/meta/argument_specs.yml:236]: Thanks, makes sense!

**Review by ja9fuchs — APPROVED** [review]
Looks good to me :+1: 

The question about the 'truthy' consistency can be taken care of separately.

**Review by rhmk — APPROVED** [review]
LGTM

**Review by rainerleber — APPROVED** [review]
LGTM

**Comment by sean-freeman** [comment]
@mr-stringer review please?

**Comment by mr-stringer** [comment]
LGTM

**Review by mr-stringer — APPROVED** [review]
LGTM

**Pull request merged**